ts/src/v8-types.ts

```diff
@@ -59,3 +59,16 @@ export interface Allocation {
 export interface LabelSet {
   [key: string]: string | number;
 }
+
+export interface GenerateAllocationLabelsFunction {
+  ({node}: {node: AllocationProfileNode}): LabelSet;
+}
+
+export interface GenerateTimeLabelsArgs {
+  node: TimeProfileNode;
+  context?: TimeProfileNodeContext;
+}
+
+export interface GenerateTimeLabelsFunction {
+  ({node, context}: GenerateTimeLabelsArgs): LabelSet;
```

Comment:
Just curious: is there an advantage to deconstructing the arg in `{node, context}` over simply `(args: GenerateTimeLabelsArgs): LabelSet`?

Reply:
Hum, no I don't see any advantage of destructuring in the interface.
I think this is just the result of me copy pasting the function args here.
I will remove it.

Reply:
Done